src/main/java/uk/gov/ch/repository/psc/PersonsWithSignificantControlRepository.java

```diff
@@ -42,6 +46,7 @@
             "listagg(noct.DESCRIPTION, +'; ') within group (order by NOC.NATURE_OF_CONTROL_TYPE_ID )    nature_of_control " +
             "from CORPORATE_BODY_APPOINTMENT CBA " +
             "inner join OFFICER_DETAIL OD on OD.OFFICER_DETAIL_ID=CBA.OFFICER_DETAIL_ID " +
+            "inner join CORPORATE_APPT_DETAIL CAD on CAD.CORPORATE_APPT_DETAIL_ID=CBA.CORPORATE_APPT_DETAIL_ID " +
```

Comment:
If CBA.CORPORATE_APPT_DETAIL_ID == null. Do all the fields it would join on also resolve to null?

Reply:
This has been updated to a left join

Reply:
So it is returning null for all the CAD fields if CBA.CORPORATE_APPT_DETAIL_ID is null?

Reply:
Yes if null - these will be optional fields. For example 00623673, MIKEES TEST 3 LIMITED, has this field as null.